js/subscribe.js

```diff
@@ -9,12 +12,48 @@ document.addEventListener('DOMContentLoaded', function () {
       document.getElementById('subscribe-error-id').style.display = "block";
     } else {
       if (validateEmail(email)) {
-        document.getElementById('email-id').classList.remove('subscription-box-input-error');
-        document.getElementById('button-box-id').classList.remove('subscription-box-button-error');
-        document.getElementById('subscribe-error-id').innerHTML = "";
-        document.getElementById('subscribe-error-id').style.display = "none";
-        document.getElementById('subscribe-result-id').style.display = "block";
-        document.getElementById("email-id").value = "";
+        grecaptcha.ready(function () {
+          try {
+            grecaptcha.execute(siteKey, { action: 'submit' }).then(function () {
+              const data = {
```

Comment:
We need to include the generated token when submitting the form.

Reply:
Tweak the code.